scripts/newinstall.sh

```diff
@@ -217,6 +214,43 @@ if true; then
 	fi
 fi
 
+##########	Test/warn about the existence of ~/.eups
+
+if true; then
+	if [[ -e ~/.eups ]]; then
+		TIMESTAMP=$(date +%Y%m%d%S)
+		cat <<-EOF
+
+		Problems have been observed when re-installing or upgrading EUPS while
+		preserving a pre-existing ~/.eups directory.  This directory primarily
+		contains cache data and is safe to remove when EUPS is not actively
```

Comment:
@ktlim / @RobertLuptonTheGood I run into this problem periodically when upgrading eups on lsstsw@lsst-dev, so I assume this is an issue for end users.  Is your preference to do nothing here or be more selective about what gets removed?